pubsub/mqtt/mqtt.go

```diff
@@ -126,6 +132,14 @@ func parseMQTTMetaData(md pubsub.Metadata) (*metadata, error) {
 		m.tlsCfg.clientKey = val
 	}
 
+	if val, ok := md.Properties[mqttBackOffMaxRetries]; ok && val != "" {
```

Comment:
Please, create doc issue to add this metadata.

Reply:
Already done here dapr/docs/issues/1267